Runtime/Tracking/Velocity/ArtificialVelocityApplier.cs

```diff
@@ -0,0 +1,131 @@
+﻿namespace Zinnia.Tracking.Velocity
+{
+    using UnityEngine;
+    using System.Collections;
+    using Malimbe.PropertySerializationAttribute;
+    using Malimbe.PropertyValidationMethod;
+    using Malimbe.XmlDocumentationAttribute;
+    using Zinnia.Extension;
+
+    /// <summary>
+    /// Applies artificial velocities to the <see cref="target"/> by changing the <see cref="Transform"/> properties.
+    /// </summary>
+    public class ArtificialVelocityApplier : MonoBehaviour
+    {
+        /// <summary>
+        /// The object to apply the artificial velocities to.
+        /// </summary>
+        [Serialized, Validated]
```

Comment:
```suggestion
        [Serialized, Validated, Cleared]
```
As this is a reference type you may want to set it to `null` via UnityEvents.

Reply:
In this instance I don't want it to be clearable. otherwise the script breaks. it _must_ have a target for it to work.

Reply:
👍